LICENSE.md

```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2015 <Your name here>
```

Comment:
<Your name here> lol - you can copy the license from one of the other blueplanet projects